pyam/core.py

```diff
@@ -745,6 +745,13 @@ def pie_plot(self, *args, **kwargs):
         ax = plotting.pie_plot(df, *args, **kwargs)
         return ax
 
+    def scatter(self, *args, **kwargs):
+        """Plot a scatter chart using metadata columns
+
+        see pyam.plotting.scatter() for all available options
+        """
+        return plotting.scatter(self.meta.reset_index(), *args, **kwargs)
```

Comment:
So this seems inconsistent with our other plotting functions. why is this `meta` by default? shouldn't we also be able to plot scatter data from the `data` df?

Reply:
The way I implemented (and used) it so far is this works for `meta` columns only. In the the unit test, I take some data and convert it to meta columns, then display those.

Assume we want to create a scatter plot between two variables over time - how would you imagine to pass over the `x` and `y` args?